tests/drivers/counter/counter_basic_api/boards/stm32h747i_disco_m7.overlay

```diff
@@ -0,0 +1,87 @@
+&timers2 {
+	st,prescaler = <239>;
+	counter {
+		status = "okay";
+	};
+};
+
+&timers3 {
+	st,prescaler = <239>;
+	counter {
+		status = "okay";
+	};
+};
+
+&timers4 {
+	st,prescaler = <239>;
+	counter {
+		status = "okay";
+	};
+};
+
+&timers5 {
+	st,prescaler = <239>;
+	counter {
+		status = "okay";
+	};
+};
+
+&timers6 {
+	st,prescaler = <239>;
+	counter {
+		status = "okay";
+	};
+};
+
+&timers7 {
+	st,prescaler = <239>;
+	counter {
+		status = "okay";
+	};
+};
+
```

Comment:
Shouldn't `timers8` be enabled?

Reply:
`timers1` and `timers8` are the Advanced-control timers and work different that driver already written